themes/default/content/registry/packages/pulumiservice/installation-configuration.md

```diff
@@ -15,7 +15,7 @@ The Pulumi Service provider is available as a package in all Pulumi languages:
 
 ## Setup
 
-To provision resources with the Pulumi Service provider, you need to have Pulumi Service credentials. Pulumi Service maintains documentation on how to create access tokens [here](https://www.pulumi.com/docs/intro/pulumi-service/accounts/#access-tokens).
+To provision resources with the Pulumi Service provider, you must have Pulumi Cloud credentials. Pulumi Cloud maintains documentation on creating [access tokens](https://www.pulumi.com/docs/intro/pulumi-cloud/accounts/#access-tokens).
```

Comment:
Should this be a relref?

Reply:
@jkodroff no, nothing should be a relref, we removed support for relrefs. but you can drop the pulumi.coms if you'd like